melodic/distribution.yaml

```diff
@@ -6269,6 +6269,16 @@ repositories:
       url: https://github.com/ros-drivers/odva_ethernetip.git
       version: indigo-devel
     status: unmaintained
+  omnibase:
+    doc:
+      type: git
+      url: https://github.com/ERC-BPGC/omnibase.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/ERC-BPGC/omnibase.git
```

Comment:
One last thing, then I think we're good; can you put a LICENSE file with the BSD license in the source repository?  I'm guessing you want to use https://opensource.org/licenses/BSD-3-Clause .  Thanks.

Reply:
I've added the BSD-3 license file in the source repository.
Thanks.